pkg/frontend/session.go

```diff
@@ -898,6 +923,26 @@ func (ses *Session) SetPrepareStmt(name string, prepareStmt *PrepareStmt) error
 			return moerr.NewInvalidState(ses.requestCtx, "too many prepared statement, max %d", MaxPrepareNumberInOneSession)
 		}
 	}
+
```

Comment:
need to clean old batch when set the same name?

if _, ok := ses.prepareStmts[name]; ok {
    ses.prepareStmts[name].close()
}

Reply:
fixed